src/data/support-title.csv

```diff
@@ -67,5 +67,8 @@
 "[召しませふぁーすとBite！]","[Try First with Bite!]"
 "[おもい、ねがう]","[A Feeling, a Wish]"
 "[Head-on fight！]","[Head-on fight!]"
+"[明日は全国的に赤でしょう♪]","[Crimson to the Entire Country Tomorrow♪]"
```

Comment:
I feel like this should be more like "Tomorrow the entire country will be dyed red" (crimson works too ig), but that would need to be shortened down of course.